docs/3.0.0-beta.x/guides/client.md

```diff
@@ -0,0 +1,159 @@
+# Setup a third party client
+
+This guide will explain how to setup a connection with a tiers client and use it everywhere in your code.
+
+In our example we will use the GitHub Node.JS client [OctoKit REST.js](https://github.com/octokit/rest.js/).
+
+This guide could also be used to setup an Axios client instance.
+
+## Installation
+
+First you will have to install the client node_module in your application by running one of the following command.
+
+:::: tabs
+
+::: tab yarn
+`yarn add @octokit/rest`
+:::
+
+::: tab npm
+`npm install @octokit/rest`
+:::
+
+::::
+
+## Create a hook
+
+To init the client, we will use [Strapi hooks system](../concepts/hooks.md). Hooks let you add new features in your Strapi application.
```

Comment:
We will use the [hook system].

I don't think we need to repeat the word Strapi ;)